node/smeshing_service_test.go

```diff
@@ -205,7 +211,70 @@ func TestSmeshingService_GRPCServices(t *testing.T) {
 	})
 }
 
+func TestSmeshingService_NonProxiedServices(t *testing.T) {
+	t.Parallel()
+	cfg := getSmeshingServiceTestConfig(t)
+	cfg.API.ProxyApiV2Address = "http://127.0.0.1:10000"
+	cfg.API.NonProxiedServices = []grpcserver.Service{
+		grpcserver.SmeshingV2Beta1,
+		grpcserver.SmeshingIdentitiesV2Beta1,
+		grpcserver.Debug,
+	}
+
+	service, err := NewSmeshingService(cfg, zaptest.NewLogger(t))
+	require.NoError(t, err)
+	defer service.Close(context.Background())
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+	require.NoError(t, service.start(ctx))
+
+	baseUrl := fmt.Sprintf("http://%s", service.apiProxy.BoundAddress)
+
+	t.Run("can reach debug service", func(t *testing.T) {
+		require.Eventually(t, func() bool {
+			url := baseUrl + "/spacemesh.v1.DebugService/ChangeLogLevel"
+			resp, err := http.Post(
+				url,
+				"application/json",
+				bytes.NewBuffer([]byte(`{"module":"grpc", "level":"debug"}`)),
+			)
+			require.NoError(t, err, url)
+			body, err := io.ReadAll(resp.Body)
+			require.NoError(t, err, url)
+			t.Logf("status: %s, body: '%s'", resp.Status, body)
+			resp.Body.Close()
+			return http.StatusOK == resp.StatusCode
+		}, time.Second*10, time.Millisecond*100)
```

Comment:
NIT: only the request needs to be repeated and I'm not sure if nested `require`s work as expected

```suggestion
		url := baseUrl + "/spacemesh.v1.DebugService/ChangeLogLevel"
		payload := bytes.NewBuffer([]byte(`{"module":"grpc", "level":"debug"}`))
		var resp *http.Response
		require.Eventually(t, func() bool {
			resp, err = http.Post(url, "application/json", payload)
			return err == nil && http.StatusOK == resp.StatusCode
		}, time.Second*10, time.Millisecond*100)
		body, err := io.ReadAll(resp.Body)
		require.NoError(t, err)
		require.NoError(t, resp.Body.Close())
		t.Logf("status: %s, body: '%s'", resp.Status, body)
```

Reply:
The proposed version won't log the response content if the request fails. I want it to log the status and the body only for debugging purposes (which is interesting when the request fails).